src/services/videos.js

```diff
@@ -41,4 +41,16 @@ export const loadSingleVideo = async (id, dispatch) => {
       payload: "Oops! Something went wrong :(",
     });
   }
+};
+
+export const getCategories = async (setCategories) => {
+  try {
+    const response = await axios.get(`${process.env.REACT_APP_URL}/categories`);
+
+    if (response.status === 200) {
+      setCategories(response?.data.categories);
+    }
+  } catch (e) {
+    console.log(e);
```

Comment:
Instead of console.log you can use alert too to get hold of what error is happening.But for now it is good!!